designer/server/src/test/scala/pl/touk/nussknacker/ui/process/deployment/DeploymentServiceSpec.scala

```diff
@@ -732,6 +773,31 @@ class DeploymentServiceSpec
     }
   }
 
+  def createDeploymentServiceWithCommentSettings(): DeploymentServiceImpl = {
```

Comment:
There is another `createDeploymentMethod` at the top of the class. We should not duplicate code.